typings/index.d.ts

```diff
@@ -3441,7 +3465,7 @@ declare module 'discord.js' {
     [K in keyof Omit<
       T,
       'client' | 'createdAt' | 'createdTimestamp' | 'id' | 'partial' | 'fetch' | 'deleted' | O
-    >]: T[K] extends (...args: any) => void ? T[K] : T[K] | null;
+    >]: T[K] extends Function ? T[K] : T[K] | null; // tslint:disable-line:ban-types
```

Comment:
Seems to have slipped through the rebase.
```suggestion
    >]: T[K] extends (...args: any) => void ? T[K] : T[K] | null;
```